src/plugins/data_table/data_table.js

```diff
@@ -2,30 +2,167 @@ var LocalDataTable = (function() {
 
   var Base = Backdraft.plugin("Base");
 
+  var DEFAULT_JST_DELIMS = {
+    evaluate    : /<%([\s\S]+?)%>/g,
+    interpolate : /<%=([\s\S]+?)%>/g,
+    escape      : /<%-([\s\S]+?)%>/g
+  };
+
+  var DataTableFilter = Base.View.extend({
+    filterTemplate : _.template('\
+      <div class="toggle-filter-button" data-toggle="dropdown">\
+        <span class="<%= filterButtonClass %>"></span>\
+      </div>\
+    ', null, DEFAULT_JST_DELIMS),
+
+    filterMenuTemplate : _.template('\
+      <div class="filterMenu dropdown-menu <%= listClass %>">\
+        <% if (filter.type === "string") { %>\
+          <input class="filter-string" id ="value" type="text" placeholder="Search <%= title %>" />\
+        <% } else if (filter.type === "numeric") { %>\
+          <ul>\
+            <li> &gt; <input id="gt" class="filter-numeric" type="text" /></li> \
+            <li> &lt; <input id="lt" class="filter-numeric" type="text"/></li> \
+            <li> = <input id="eq" class="filter-numeric" type="text" /></li> \
+          </ul>\
+        <% } else if (filter.type === "list") { %>\
+          <ul>\
+            <% _.each(filter.options, function(element, index) { %>\
+              <li>\
+                <label>\
```

Comment:
I would rather not have an ID for each checkbox element if just for the label here..  besides the extra code and work, there's a chance of not actually having unique IDs for the entire DOM on page (which would be a worse HTML bug IMO).  And how you wrapped it here works for me (since there is only one input within the `label` I don't see any ambiguity and the browser is going to have the same behavior here).